standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/ServletSecurity.java

```diff
@@ -85,8 +85,10 @@ public class ServletSecurity {
   private final Configuration conf;
   private JWTValidator jwtValidator = null;
 
-  public ServletSecurity(Configuration conf) {
-    this(conf, isAuthJwt(conf));
+  public ServletSecurity(String authType, Configuration conf) {
+    this.conf = conf;
```

Comment:
@okumin, why not call the existing constructor? 
````
this(conf, isAuthJwt(authTypeStr));
````

Reply:
Just my bad. Updated!